internal/deviations/deviations.go

```diff
@@ -597,14 +597,14 @@ func ISISCounterPartChangesUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIsisCounterPartChangesUnsupported()
 }
 
-// TransceiverThresholdsUnsupported returns true if the device does not support threshold container under /components/component/transceiver.
-// Default value is false.
-func TransceiverThresholdsUnsupported(dut *ondatra.DUTDevice) bool {
-	return lookupDUTDeviations(dut).GetTransceiverThresholdsUnsupported()
+// TunnelStatePathUnsupported returns true for devices that require configuring
+// /interfaces/interface/state/counters/in-pkts, in-octets,out-pkts, out-octetsis not supported.
+func TunnelStatePathUnsupported(dut *ondatra.DUTDevice) bool {
+	return lookupDUTDeviations(dut).GetTunnelStatePathUnsupported()
 }
 
-// InterfaceLoopbackModeRawGnmi returns true if interface loopback mode needs to be updated using raw gnmi API due to server version.
-// Default value is false.
-func InterfaceLoopbackModeRawGnmi(dut *ondatra.DUTDevice) bool {
```

Comment:
What do you want to do about the existing test using this deviation?
https://github.com/openconfig/featureprofiles/blob/24d10056e7453a2750439d7a329a4c02ba838616/feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go#L231